Circle CI crashes when encounters html-noplot rule

see [here](https://circleci.com/gh/pydicom/pydicom/58?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

here is the complete output - basically (I don't think) the rule exists yet in the makefile for the doc (in the doc folder)

```
  Stored in directory: /home/ubuntu/.cache/pip/wheels/b8/09/cf/9801757ff42c85e23678e2520f7dee38f3fdd6455f7550f676
Successfully built sphinx-gallery
Installing collected packages: sphinx-gallery
Successfully installed sphinx-gallery-0.1.11
+ python setup.py develop
running develop
running egg_info
creating pydicom.egg-info
writing pydicom.egg-info/PKG-INFO
writing dependency_links to pydicom.egg-info/dependency_links.txt
writing top-level names to pydicom.egg-info/top_level.txt
writing manifest file 'pydicom.egg-info/SOURCES.txt'
reading manifest file 'pydicom.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'pydicom.egg-info/SOURCES.txt'
running build_ext
Creating /home/ubuntu/miniconda/envs/testenv/lib/python3.6/site-packages/pydicom.egg-link (link to .)
Adding pydicom 1.0.0a1 to easy-install.pth file

Installed /home/ubuntu/pydicom
Processing dependencies for pydicom==1.0.0a1
Finished processing dependencies for pydicom==1.0.0a1
+ set -o pipefail
+ cd doc
+ tee /home/ubuntu/log.txt
+ make html-noplot
make: *** No rule to make target `html-noplot'.  Stop.

./build_tools/circle/build_doc.sh returned exit code 2

Action failed: ./build_tools/circle/build_doc.sh
```

--- a/Makefile
+++ b/Makefile
@@ -17,9 +17,6 @@ test: test-code test-doc
 doc:
 	make -C doc html
 
-doc-noplot:
-	make -C doc html-noplot
-
 clean:
 	find . -name "*.so" -o -name "*.pyc" -o -name "*.md5" -o -name "*.pyd" -o -name "*~" | xargs rm -f
 	find . -name "*.pyx" -exec ./tools/rm_pyx_c_file.sh {} \;

--- a/build_tools/circle/build_doc.sh
+++ b/build_tools/circle/build_doc.sh
@@ -78,7 +78,7 @@ then
     MAKE_TARGET="dist LATEXMKOPTS=-halt-on-error"
 elif [[ "$build_type" =~ ^QUICK ]]
 then
-	MAKE_TARGET=html-noplot
+	MAKE_TARGET=html
 else
     MAKE_TARGET=html
 fi

